l_display.py

```diff
@@ -0,0 +1,295 @@
+import tkinter as tk
+from tkinter import ttk
+from PIL import Image, ImageTk, ImageDraw
+import lxml.etree as ET
+import os
+
+
+def create_gui():
+    """
+    Create the graphical user interface (GUI) for editing MEI files.
+
+    This function creates a GUI with an image display, text fields for editing,
+    and buttons for navigating and saving changes.
+    """
+    mei_folder = "Liber Usualis - mei5"
+    image_folder = "YOUR FOLDER HERE"
+    index = 1
+
+    # Find the first existing image file
+    while True:
+        image_file = f"liber_{index:04d}.jpg"
+        image_path = os.path.join(image_folder, image_file)
+        if os.path.exists(image_path):
+            break
+        index += 1
+        if index > 2340:  # assume max 9999 images
+            print("No image files found!")
+            return
```

Comment:
What's going on with the numbers in the `if` clause and the comment? Should they be different? Why not just check if there are any `*.jpg` files in the given folder?